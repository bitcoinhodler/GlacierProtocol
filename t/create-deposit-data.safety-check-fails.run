#!/bin/bash

$GLACIERSCRIPT --regtest=$1 create-deposit-data -m 3 -n 5 << INPUT 2>&1
y
y
n
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected glacierscript.py to fail"
    exit 1
fi
