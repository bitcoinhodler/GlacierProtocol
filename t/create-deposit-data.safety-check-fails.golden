Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?
 Safety check failed. Exiting.
